### ISSUE TITLE
Support upcoming v0.5 OME-NGFF standard

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-tasks-core#168]
Let's make sure we support the upcoming OME-NGFF v0.5 standard. This is have 2 major parts:
1. The AnnData tables will be an official part of the spec, with some details on what metadata needs to be saved with AnnData tables (that I think we don't fully cover yet). It's not been merged into the spec yet, WIP is here: https://github.com/ome/ngff/pull/64
2. Work on coordinate transformations, coordinate spaces, making OME-Zarrs N-dimensional etc. There are multiple PRs on this and I also don't have the best overview on it yet.

Maybe there are also updates on general metadata storage, but I don't have a good overview about this yet. There is transitional OMERO metadata added in the XML format, but that doesn't seem to be a long-term solution, see https://ngff.openmicroscopy.org/latest/#bf2raw

I'll try to get a better feeling of the timeline for an 0.5 release of the spec to see whether some of the other issues in this milestone should be tackled first or only after adapting to 0.5

Tasks:
- [ ] Our AnnData tables follow the specification
- [ ] We follow all the coordinate space

**Comment by tcompa** [comment]
Something to keep an eye on: https://github.com/JaneliaSciComp/pydantic-ome-ngff.

I'm not sure it'll be useful for us, but it could - and the upgrade of fractal-tasks-core to ngff-0.5 support would be a good moment to evaluate this upcoming project.

**Comment by tcompa** [comment]
> Let's make sure we support the upcoming OME-NGFF v0.5 standard. This is have 2 major parts:

v0.5 likely won't include either of the expected features (tables and transforms), but it will adopt zarr v3 (https://forum.image.sc/t/ome-ngff-update-postponing-transforms-previously-v0-5/95617).

Closing this one, in favor of #294
